### PR TITLE
initial setup of holochain/dev-resources-kit

### PR DESCRIPTION
**Pull request by d3sbar**
some initial content and directory structure for holochain/dev-resources-kit, includes:
* starting readme, awesome files;
* placeholders for faqs and media collateral;
* starting documentation set for holodoc tutorial project;

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+* Common setup and toolchain guidance for developers of all levels that has been tested by the community and \n
+  is known to work;
+* Tutorial projects that allow step-by-step understanding of how to build a hApp and contribute to either hApps \n
+  or the core Holochain project;
+* Example applications that explore all the aspects of the Holochain technology and serve a useful purpose for \n
+  the community and stated education objectives;
+
+## Resources
+* See [Awesome-Holochain](awesome.md)
+* See [Holochain Developer Portal](https://dev.holochain.org)
```
Comment (Connoropolous) [inline README.md]: https://developer.holochain.org :)

Comment (d3sbar) [reply README.md]: 👍 got it. besides that edit - any problems merging this upstream @Connoropolous @dhtnetwork

Comment (dhtnetwork) [reply README.md]: > 👍 got it. besides that edit - any problems merging this upstream @Connoropolous @dhtnetwork

My only request. Can we consider changing the name to holochaindoc? Global name change request. We want people to be clear that it's Holochain and not  @Holo.  
projects/holochaindoc/specs/index.md updated from

Comment (Connoropolous) [reply README.md]: Agreed, that'd be better

**Comment by d3sbar** [comment]
I am going to do a new PR with a cleaned up set of files.

**Pull request closed without merging**